_docs/scripts/linkedHeadings.js

```diff
@@ -0,0 +1,3 @@
+$('h2, h3').filter('[id]').each(function () {
```

Comment:
Currently, the cursor does become a pointer despite the screencasts attempts to hide that.
![tags](https://cloud.githubusercontent.com/assets/2326518/19784471/ed06a1b4-9c63-11e6-972e-ad4352da1e3b.jpg)

For consistency, we could make the `h1` and `h4` also clickable - will do.